### DISCUSSION TITLE
Failed to add PR: expected head oid does not match current head oid

### DISCUSSION THREAD

**Discussion by hfhbd** [community/community#59365] [Mobile] [answered]
### Select Topic Area

Bug

### Body

https://github.com/community/community/assets/22521688/815ed0b3-e619-464a-bfc8-943aeac086dc

Reproducer:
- Have an outdated PR
- Click `update branch`
- Click `auto-merge`

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.118.1 (103027020)
Device: iPhone 12 mini (iOS 17.0)
</details>

**Comment by stevepopovich** [comment, marked as answer]
Hi @hfhbd thanks for the report! Does this go away after refreshing? I believe that if you update branch then pull to refresh, things should work.

Either way, this is a bug that we should look at. Thanks!

**Reply by hfhbd** [reply]
Yes, manually refreshing does work, as a workaround.

**Reply by stevepopovich** [reply]
Cool, will file this bug. Thanks!

**Reply by Mhmoud0316** [reply]
https://github.com/facebook/facebook-ios-sdk/releases/download/v16.1.2/FBAEMKit-Static_XCFramework.zip

**Reply by Mhmoud0316** [reply]
> https://github.com/facebook/facebook-ios-sdk/releases/download/v16.1.2/FBAEMKit-Static_XCFramework.zip
